### PR TITLE
Switch to optional upsert when loading specifications

### PR DESCRIPTION
**Pull request by eacharles**

### CODE REVIEW

File: src/lsst/cmservice/handlers/functions.py [hunk 1]
```diff
@@ -132,30 +154,43 @@ async def create_script_template(
...
-        name=key,
-        file_path=config_values["file_path"],
-    )
+    if script_template is None:
+        return await ScriptTemplate.load(
+            session,
+            name=key,
+            file_path=config_values["file_path"],
+        )
+    full_file_path = os.path.abspath(os.path.expandvars(config_values["file_path"]))
```
Comment (fritzm) [inline src/lsst/cmservice/handlers/functions.py:178]: Asymmetry with file handling here, vs. the load case.  Is there a good reason?

**Review by fritzm — APPROVED** [review]
Looks good to me -- asymmetric file handling for load/update in one place that would be nice to fix if possible.

File: src/lsst/cmservice/handlers/functions.py (hunk 1 above)
Comment (eacharles) [reply src/lsst/cmservice/handlers/functions.py]: 🤓

Comment (eacharles) [reply src/lsst/cmservice/handlers/functions.py]: I was lazy & didn't want to right a new function.

**Comment by eacharles** [comment]
Found a jira issue.   Gonna close this PR and open with the correctly named branch

**Pull request closed without merging**